tests/integration/test_tile.py

```diff
@@ -21,7 +21,7 @@ class Describe_Tile:
         ),
     )
     def it_knows_if_is_is_almost_white(self, tile_img, expected_result):
-        coords = CoordinatePair(0, 512, 0, 512)
+        coords = CP(0, 512, 0, 512)
```

Comment:
Why shortening the name? To me is not clear what a CP is

Reply:
`CoordinatePair` is `CP` only in the tests. I'd prefer a short name here to groom the fixtures TBH